### PR TITLE
2d curve segmentationk - initial interfaces

### PR DESCRIPTION
**Pull request by JozoUrban**

### CODE REVIEW

File: ImageInLib/ImageInLib/include/segmentation.h
Comment (KonanAl) [inline ImageInLib/ImageInLib/include/segmentation.h:1]: This file is also modified in my pull request

File: ImageInLib/ImageInLib/src/segmentation.c
Comment (KonanAl) [inline ImageInLib/ImageInLib/src/segmentation.c:1]: Also this, because I added 2d subsurf and gsubsurf

**Review by KonanAl — APPROVED** [review]

File: ImageInLib/ImageInLib/include/segmentation.h
Comment (JozoUrban) [reply ImageInLib/ImageInLib/include/segmentation.h:1]: Do not worry. It is standard issue, we will overcome easily.
It means, that we will need to do merge. But not now.
If your changes will be in master earlier than mine, I will do merge from master -> this branch.

**Comment by KonanAl** [comment]
My pull request is on the branch "update-inflanet" not on "master" branch. So, can I merge your changes ?
________________________________
De : JozoUrban ***@***.***>
Envoyé : mardi 11 juillet 2023 10:59
À : ImageInLib/ImageInLib ***@***.***>
Cc : Konan Allaly ***@***.***>; Review requested ***@***.***>
Objet : Re: [ImageInLib/ImageInLib] 2d curve segmentationk - initial interfaces (PR #73)


@JozoUrban commented on this pull request.

________________________________

On ImageInLib/ImageInLib/src/segmentation.c<https://github.com/ImageInLib/ImageInLib/pull/73#discussion_r1259435141>:

DO not worry, we will merge it

**Pull request merged**